README.md

```diff
@@ -393,14 +393,13 @@ var name = 'ubuntu-http';
 zone.createVM(name, { os: 'ubuntu' }, function(err, vm, operation) {
   // `operation` lets you check the status of long-running tasks.
 
-  operation.on('complete', function(err, metadata) {
-    if (!err) {
-      // Virtual machine created!
-    }
+  operation.on('complete', function(metadata) {
+    // Virtual machine created!
   });
 });
 ```
 
+You can read more about opeation events [here](https://github.com/GoogleCloudPlatform/gcloud-node/blob/276c847669470c3639eb78d7d87a09db7a9b5c18/lib/compute/operation.js#L83).
```

Comment:
You can link to the more readable version here: https://googlecloudplatform.github.io/gcloud-node/#/docs/compute/operation